docs/DeveloperGuide.md

```diff
@@ -617,6 +617,28 @@ person on the list is born before 12-12-2020.
 
 ### Adding a friend group: `add-group`
 
+Prerequisites: List all person using the list command. There is at least a person present in the list. 
```

Comment:
First line contradicts the 2nd. There should be at least 3 people in the list? I would do the test cases with just 2 people so it would be easier to follow.